addons/binding/org.openhab.binding.minecraft/README.md

```diff
@@ -0,0 +1,46 @@
+# Openhab-Minecraft-Binding
```

Comment:
note that the correct spelling is "openHAB" - would be good if you could replace all occurrences.

In this location here, "Minecraft Binding" should be fine anyhow.